libs/prelude/Language/Reflection.idr

```diff
@@ -396,9 +396,9 @@ mutual
   withNInj Refl = (Refl, Refl)
 
   private
-  instanceNInj : (InstanceN n xs = InstanceN n' ys) -> (n = n', xs = ys)
-  instanceNInj Refl = (Refl, Refl)
-
+  implementationNInj : (ImplementationN n xs = ImplementationN n' ys) -> (n = n', xs = ys)
```

Comment:
@jfdm I am not sure what the policy is. But I guess it is OK, since they're private?